app/human-detector-app/package.json

```diff
@@ -9,7 +9,7 @@
     "web": "expo start --web",
     "eject": "expo eject",
     "test": "jest",
-    "testBDD": "jest LoggingIn.steps.js PushNotification.steps.js",
+    "testBDD": "jest LoggingIn.steps.tsx PushNotification.steps.tsx",
```

Comment:
super nitpick: `tsx` files are for JSX (usually React components), these should be plain `ts` files

Reply:
Addressed in bc61c9dfd9b3d2d36aa2c2bb13b3aa3a537e662a